sites-available/ssl.no-default

```diff
@@ -6,7 +6,9 @@
 # returning 444 "no response".
 
 server {
+  listen [::]:443 ssl default_server;
```

Comment:
Yes, it should probably be the same. And I'd argue to include it in both. While the difference will probably not be noticeable except from very high loads, it's the right way to do and should not produce any harm. Maybe on some systems where `TCP_DEFER_ACCEPT` is not accepted. But these are rare edge cases and these boilerplates are not intended do fit everywhere, but provide an overall sane starting point.